dags/main_summary.py

```diff
@@ -259,18 +195,17 @@
     email=["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"],
     dag=dag)
 
+wait_for_clients_daily = ExternalTaskSensor(
+    task_id="wait_for_clients_daily",
+    external_dag_id="bqetl_clients_daily",
+    external_task_id="telemetry_derived__clients_daily__v6",
+    dag=dag)
+
```

Comment:
Having an external task sensor here for clients_daily now feels very strange. I think within the scope of this PR, we should break out these two tasks that depend on clients_daily.

`experiments_daily_active_clients` should probably be in its own DAG generated by bigquery-etl. 

And then the clients_daily export should probably also be its own DAG. Or perhaps we should call the DAG main_summary_export, and include the main_summary_export task there, too.

Reply:
https://github.com/mozilla/bigquery-etl/pull/1094 moves queries out of `main_summary`. I named the export DAG `parquet_export` for now, but I'm open to alternative names too.